MapboxCoreNavigation/RouteProgress.swift

```diff
@@ -431,6 +431,11 @@ open class RouteStepProgress: NSObject {
         return (1 - fractionTraveled) * step.expectedTravelTime
     }
     
+    /**
+     Coordinates from current location to end of current step
+     */
+    @objc public var coordinatesRemaining: [CLLocationCoordinate2D] = []
```

Comment:
@d-prukop this should be an optional property. This will help determine the initial state vs. the final state. Basically, how could we detect that the `coordinatesRemaining` has not been set as opposed to when we slice a polyline and get an empty array in return?